insta-demo/pulp-insta-demo.sh

```diff
@@ -93,7 +93,15 @@ else
   cd pulp-operator-$BRANCH || failure_message
 fi
 
-sudo .travis/k3s-install.sh --insta-demo || failure_message
+# Check if k3s is already installed
+if command -v kubectl > /dev/null; then
+  echo "k8s already installed!"
+elif [ -x /usr/local/bin/kubectl ]; then
+  echo "k8s already installed!"
+else
+    sudo .travis/k3s-install.sh --insta-demo || failure_message
+fi
+
```

Comment:
I am reluctant to even try to support the scenario of other Kubernetes implementations in pulp-insta-demo. I feel like this is a design decision we have to make.

For one thing, the uninstall feature will no longer work. I suppose we could keep track of that in a variable in this script, and not print out the uninstall message at the end.

For another thing, we use a RWO (ReadWriteOnce) volume for up.sh:
https://github.com/pulp/pulp-operator/blob/master/deploy/crds/pulpproject_v1alpha1_pulp_cr.travis.yaml
But if the cluster has more than 1 node, then RWO will fail.

I think we should just exit if `kubectl` is already present. And do the check for `kubectl` at the beginning of the script.

Reply:
this is a hack I had to use, as I'm installing collections at Dockerfile, the image built during the test works, but if it pulls from quay, it fails!

Reply:
I see. Just remove it at the end.

Reply:
I don't follow what you meant by the "end"?
Remove it before merging the PR?

Reply:
Yes, that's what I  meant.